developer_docs/installation.md

```diff
@@ -80,3 +80,10 @@ name, you can also set it using this variable. I.e.:
 For more information on using a custom domain, see this documentation link:
 
 http://docs.aws.amazon.com/AmazonS3/latest/dev/VirtualHosting.html#VirtualHostingCustomURLs
+
+## Github API Configuration
+
+This is an optional part, unless you are working on the part of the project that requires you to work with Github API (i.e: login with your Github Account, make request to Github API to download example sketches,...). An overview about Github API can be found [here](https://developer.github.com/v3/)
```

Comment:
I think it would make sense to list the specific things that the GitHub credentials are necessary for:
- Authentication with GitHub
- Importing the p5.js examples to your local database
- Rendering the 404 pages

Which you've already done! But I think this language could be changed to be something like, "If you are working on a part of the application that needs X, then you will need to get GitHub API credentials.